src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+    
     Version 2.0
-
-    The primary goals of this format is to allow a simple XML format
-    that is mostly human readable. The generation and parsing of the
-    various data types are done through the TypeConverter classes
+    
+    The primary goals of this format is to allow a simple XML format 

```

Comment:
```suggestion
    The primary goal of this format is to allow a simple XML format 

```

Reply:
Autogenerated by VS. Can't fix; it will rewrite it next time it handles it.

Reply:
Can someone file a bug against VS? :-)

Reply:
I've already started the thread :smile: